src/zenml/zen_server/zen_server_api.py

```diff
@@ -11,18 +11,28 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
-"""Zen Server API."""
+"""Zen Server API.
+
+To run this file locally, execute:
+
+    ```
+    uvicorn zenml.zen_server.zen_server_api:app --reload
+    ```
+"""
 
 import os
 from asyncio.log import logger
 from genericpath import isfile
+from time import sleep
 from typing import Any, List
 
+from anyio import to_thread
 from fastapi import FastAPI, HTTPException, Request
 from fastapi.exceptions import RequestValidationError
 from fastapi.responses import ORJSONResponse
 from fastapi.staticfiles import StaticFiles
 from fastapi.templating import Jinja2Templates
+from py import log
```

Comment:
This and the `sleep` import seem unused?

Reply:
Correct, I removed them. The linter was also flagging them.